### PR TITLE
[Doc] Update global settings

### PR DESCRIPTION
**Pull request by Olinaaaloompa**
Editorial updates

### CODE REVIEW

File: docs/lang/articles/reference/global_settings.md [hunk 1]
```diff
@@ -142,28 +141,28 @@ Below are some environment variables that you can set to customize your Taichi p
...
 
 :::
 
 ## Compilation
 
-- To disable advanced optimization to save compile time and reduce possible errors: `ti.init(advanced_optimization=False)`.
-- To disable fast math to prevent possible undefined math behavior: `ti.init(fast_math=False)`.
-- To print intermediate IR generated: `ti.init(print_ir=True)`. Note that compiled kernels are [cached by default](../performance_tuning/performance.md#offline-cache). To force compilation and IR emission, use `ti.init(print_ir=True, offline_cache=False)`.
+- To disable advanced optimization: `ti.init(advanced_optimization=False)`, which helps save compile time and reduce possible errors.
+- To disable fast math: `ti.init(fast_math=False)`, which helps prevent possible undefined math behavior
```
Comment (Olinaaaloompa) [inline docs/lang/articles/reference/global_settings.md:164]:
```suggestion
- To disable fast math: `ti.init(fast_math=False)`, which helps prevent possible undefined math behavior.
```

File: docs/lang/articles/reference/global_settings.md [hunk 2]
```diff
@@ -183,20 +182,19 @@ If you want to use CUDA and Taichi's GGUI system at the same time on a machine w
...
-- To cache compiled runtime bitcode in **dev mode** to save startup time: `export TI_CACHE_RUNTIME_BITCODE=1`.
+- To cache compiled runtime bitcode in **dev mode**: `export TI_CACHE_RUNTIME_BITCODE=1`, which saves startup time.
 - To allocate four threads to run a test: `export TI_TEST_THREADS=4` or `python tests/run_tests.py -t4`.
 
 
 :::note
 
-If `ti.init` is called twice, then the configuration in first invocation
-will be completely discarded, e.g.:
+If `ti.init` is called twice, the configuration in the first call is discarded. for example:
```
Comment (Olinaaaloompa) [inline docs/lang/articles/reference/global_settings.md:197]:
```suggestion
If `ti.init` is called twice, the configuration in the first call is discarded. For example:
```

File: docs/lang/articles/reference/global_settings.md [hunk 3]
```diff
@@ -4,19 +4,17 @@ sidebar_position: 3
 
 # Global Settings
 
+Every Taichi program starts with `ti.init()`. You can customize your Taichi runtime by passing arguments to `ti.init()` or setting environment variables. Each argument or environment variable controls one specific behavior of the Taichi runtime. For example, the argument `arch` specifies the backend, and the argument `debug` decides whether to run the program in debug mode.
 
-The call `ti.init()` is indispensable in every Taichi program. Taichi allows you to customize your Taichi runtime by passing arguments to `ti.init()` or setting environment variables. Each argument or environment variable controls one specific behavior of the Taichi runtime. For example, the argument `arch` specifies the backend, and the argument `debug` decides whether to run the program in debug mode. The document [Hello, World!](../get-started/index.md) gives a brief introduction to this function, and this document provides more details.
+When you call `ti.init()`, Taichi executes the following steps to initialize a specific configuration. We use the `arch` argument as an example:
```
Comment (Vissidarte-Herman) [inline docs/lang/articles/reference/global_settings.md:9]: "executes the following steps" is not grammatical. 
```suggestion
Taichi executes the following to initialize a specific configuration in the `ti.init()` method call. Taking the `arch` argument as an example:
```

File: docs/lang/articles/reference/global_settings.md [hunk 4]
```diff
@@ -85,19 +83,19 @@ Following are some frequently used configurations that the `ti.init()` method ca
...
     verbose: bool
-        Turn on/off verbose outputs. For example, set `ti.init(verbose=False)` to eliminate verbose outputs.
+        Turn on/off verbose outputs. For example, `ti.init(verbose=False)` prevents verbose outputs.
 
 
 [Develop Options]
 
     gdb_trigger: bool
-        Enable/disable triggering GDB when Taichi crashes, e.g. `ti.init(gdb_trigger=True)`.
+        Enable/disable GDB when Taichi crashes. For example, `ti.init(gdb_trigger=True)` activates GDB.
```
Comment (Vissidarte-Herman) [inline docs/lang/articles/reference/global_settings.md:92]: Avoid using two different terms for the same meaning. Use enables instead.

**Review by Vissidarte-Herman — CHANGES_REQUESTED** [review]

**Pull request merged**